common/parser.hpp

```diff
@@ -194,50 +197,37 @@ struct commands_xml
     std::map<std::string, custom_formatter> custom_formatters;
 };
 
-inline std::string hexify(unsigned char n)
+// While working from files, convert the file content into recognized raw format
+inline void preprocess_data_payload(std::vector<std::string>& params)
```

Comment:
Like I said, I don't like the name...
Maybe `filename_argument_to_raw_data()`?

Reply:
Renamed to`file_argument_to_blob`